trunk/SUAVE/Methods/Missions/Segments/Cruise/Constant_Speed_Linear_Altitude.py

```diff
@@ -0,0 +1,35 @@
+# ----------------------------------------------------------------------
```

Comment:
Missing header and docstring.

Reply:
I actually removed this file. It is in the wrong folder